go/chat/helper.go

```diff
@@ -97,24 +97,14 @@ func (s *sendHelper) conversation(ctx context.Context) error {
 		return err
 	}
 	uid := gregor1.UID(kuid.ToBytes())
-
-	oneChatPerTLF := true
-	convs, _, err := FindConversations(ctx, s.G(), s.DebugLabeler, s.remoteInterface, uid, s.canonicalName,
-		chat1.TopicType_CHAT, s.membersType, chat1.TLFVisibility_PRIVATE, s.topicName, &oneChatPerTLF)
+	conv, _, err := NewConversation(ctx, s.G(), uid, s.canonicalName, s.topicName,
```

Comment:
If `NewConversation` gets called on the same specs as one that already exists, it just returns that one. The call to `FindConversations` in the beginning handles that. If we try to send to a TLF ID and topic type that already has a conversation (and we set a topic name), and we are in KBFS mode, then Gregor returns a special error (with the existing conversation), but then `NewConversation` will just return that conversation with no error.

So it is safe to repeatedly call it.